client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -140,29 +151,70 @@ export const RequestLineEditForm = ({
       }
       Middle={
         <>
+          {isPacksEnabled && (
+            <Box display="flex" justifyContent="flex-end" alignItems="center">
+              <Switch
+                label={t('label.units')}
+                checked={isPacks}
+                onChange={(_event, checked) => setIsPacks(checked)}
+                size="small"
+              />
+              <Box paddingLeft={2} paddingRight={2}>
+                {t('label.packs')}
+              </Box>
+            </Box>
+          )}
           <InputWithLabelRow
             Input={
               <NumericTextInput
-                width={150}
+                width={100}
                 value={Math.round(draftLine?.suggestedQuantity ?? 0)}
                 disabled
               />
             }
-            labelWidth="150px"
-            label={t('label.suggested-quantity', { ns: 'replenishment' })}
+            labelWidth="750px"
+            label={t(
+              isPacksEnabled
+                ? 'label.suggested-units'
```

Comment:
i like `suggested-units`, is this term that we currently use in ui or just in this case (vs quantity) ?